var/spack/repos/builtin/packages/emacs/package.py

```diff
@@ -90,10 +91,13 @@ def configure_args(self):
         else:
             args = ["--without-x"]
 
-        # On OS X/macOS, do not build "nextstep/Emacs.app", because
-        # doing so throws an error at build-time
         if sys.platform == "darwin":
-            args.append("--without-ns")
+            if "+gui" in spec:
```

Comment:
```suggestion
            if spec.satisfies("+gui"):
```

It's now recommended to use `spec.satisfies()` instead of `in`.